src/graphics/internal_canvas.rs

```diff
@@ -407,6 +414,15 @@ impl<'a> InternalCanvas<'a> {
                 z: param.color.b,
                 w: param.color.a,
             },
+            scale: if scale {
+                glam::Vec2::new(
+                    instances.image.width() as f32,
+                    instances.image.height() as f32,
+                )
+            } else {
+                glam::Vec2::ZERO
+            }
+            .into(),
```

Comment:
And then please add a comment like "here we pass an additional scale parameter, which we need as we passed None as image_scale above"